### PR TITLE
[bug 1087676] Pin articles to the top

### PR DESCRIPTION
**Pull request by rehandalal**
Just realized I never actually filed this! D'oh!!

r?

### CODE REVIEW

**Comment by mythmon** [comment]
Travis :boom:

**Comment by rehandalal** [comment]
fixed ^

File: kitsune/products/tests/test_templates.py [hunk 1]
```diff
@@ -93,10 +93,9 @@ def test_document_listing_order(self):
             doc.products.add(p)
             docs.append(doc)
 
-        # Add a helpful vote to the second document. It should be first now.
-        rev = docs[1].current_revision
-        helpful_vote(revision=rev, helpful=True, save=True)
-        docs[1].save()  # Votes don't trigger a reindex.
+        # Add a lower display order to the second document. It should be first now.
```
Comment (mythmon) [inline kitsune/products/tests/test_templates.py]: Can you tweak the docstring of this test function to mention the display_order field too?

**Comment by mythmon** [comment]
More :boom:

**Comment by rehandalal** [comment]
gah! should be fixed for realz ^

**Comment by mythmon** [comment]
Nope

**Comment by rehandalal** [comment]
if this doesn't fix it, i'll eat my :tophat: ^

**Comment by rehandalal** [comment]
@mythmon is this good to go now that travis is happy?

**Comment by mythmon** [comment]
lgtm

**Pull request merged**